src/utils/allocators.zig

```diff
@@ -655,6 +655,24 @@ pub const DiskMemoryAllocator = struct {
     inline fn fileNameBounded(file_index: u32) FileNameFmtSpec.BoundedArray(struct { u32 }) {
         return FileNameFmtSpec.fmt(.{file_index});
     }
+
+    /// helper function which is simpler than the allocator interface
+    pub fn allocFilename(self: *const Self, file_name: []const u8, n: u64) ![]u8 {
+        const file = try self.dir.createFile(file_name, .{ .read = true, .truncate = true });
+        defer file.close();
+
+        try file.setEndPos(n);
+
+        const memory = std.posix.mmap(
+            null,
+            n,
+            std.posix.PROT.READ | std.posix.PROT.WRITE,
+            std.posix.MAP{ .TYPE = .SHARED },
+            file.handle,
+            0,
+        );
+        return memory;
+    }
```

Comment:
Did this get left in from the previous stack item?

Reply:
yeah it was - thanks - aa1abd1f